tests/testthat/test_aggregate.R

```diff
@@ -130,7 +129,48 @@ test_that("aggregation: aggregate_by_vector", {
                  aggregate_by_vector(x, k_fact, colSums)[same_row_names, ])
     expect_equal(x2_sum_expected[same_row_names, ],
                  aggregate_by_vector(x, k_fact2, colSums)[same_row_names, ])
-
+    
```

Comment:
would it maybe make sense to test also on `Matrix`?

Reply:
Sure, `Matrix` would make sense as well, although we are not very interested in this feature for `QFeatures` application (cf: https://github.com/rformassspectrometry/QFeatures/issues/157). Do you want me to provide compatibility for `Matrix` objects as well?

Reply:
It was more just to have also a unit test that runs on systems that don't have HDF5Array available - sort as a proof of working functionality. Makes however only sense if the HDF5Array extends `Matrix`. If not, just ignore this comment.

Reply:
Ok, I misunderstood your question. I don't think this would work since `inherits(hdf5MatrixObject, "Matrix")` returns false so it does not extend `Matrix`. But I get your point and I can't think of a solution that would test the chunks in the absence of `HDF5Array`. The only improvement I could see is to use `testthat::skip_if()` to at least throw a message that the chunk was not tested. What do you think?

Reply:
Ah, no, then ignore my comment. All good.